### DISCUSSION TITLE
Is there a place where you have videos on how to use VSC?

### DISCUSSION THREAD

**Discussion by dhfreinkel** [codefori/vscode-ibmi#1055] [Q&A]
I have been using VSC now for a while and I do like it.

However, as you add functionality, I find it difficult to find videos on its use.

Is there a single location that contains all videos that I can go to and select those I want to view. Example, how to use the new debugger.

TIA.

**Comment by worksofliam** [comment]
Hi!

This is the place I collect videos: https://www.youtube.com/playlist?list=PLNl31cqBafCp-ml8WqPeriHWLD1bkg7KL

I am yet to create a video on debugging.

**Reply by worksofliam** [reply]
Video has been created: https://www.youtube.com/watch?v=ljphu1_nTVg

**Comment by dhfreinkel** [comment]
Thank you.

 

Darryl Freinkel

A4G

Telephone: 770.321.8562 Mobile: 678.355.8562
